tests/helpers/main.ts

```diff
@@ -82,26 +91,20 @@ const requireExtractedFiles = async function (files: FunctionResult[]): Promise<
   expect(jsFiles.every(Boolean)).toBe(true)
 }
 
-export const unzipFiles = async function (files: FunctionResult[], targetPathGenerator?: (path: string) => string) {
-  // unzip functions in series, as on windows it sometimes fails with permission
-  // errors if two unzip calls try to create the same file
-  for (const { path } of files) {
-    await unzipFile({ path, targetPathGenerator })
-  }
+export const unzipFiles = async function (files: FunctionResult[]): Promise<TestFunctionResult[]> {
+  await Promise.all(
+    Object.keys(files).map(async (key) => {
+      const { path, name } = files[key]
+      const dest = join(dirname(path), name)
+      // eslint-disable-next-line no-param-reassign
+      files[key].unzipPath = await unzipFile(path, dest)
```

Comment:
We add the unzipped path to the functions array, so we know in tests where the tests were extracted to.